### PR TITLE
Improved comparison mode and added summary button

### PR DESCRIPTION
**Pull request by airenzp**
## Description

During comparison mode hided legends and tables. Kept filters but relocated them to below the plot
Also Profile summary use case is now drafted, we can select a domain and open the plot (empty for now). See corresponding PR in sjpp
Screenshot 2024-07-25 at 3 07 37 PM

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [x] Tests: added and/or passed unit and integration tests, or N/A
- [x] Todos: commented or documented, or N/A
- [x] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

File: client/termdb/tree.js [hunk 1]
```diff
@@ -201,8 +201,10 @@ class TdbTree {
 					copy.terms = t0.terms
 				}
 			}
+
 			this.termsById[copy.id] = copy
 		}
+		if (terms) for (const child of terms) child.parent = term
```
Comment (xzhou82) [inline client/termdb/tree.js:207]: since this affects all mass plots, please comment on purpose

File: server/shared/termdb.usecase.js [hunk 2]
```diff
@@ -100,7 +100,13 @@ export function isUsableTerm(term, _usecase, ds) {
 				if (graphableTypes.has(term.type)) uses.add('plot')
 				if (!term.isleaf) uses.add('branch')
 			}
-
+		case 'profile':
+			if (!term.isleaf) {
```
Comment (xzhou82) [inline server/shared/termdb.usecase.js:104]: please comment on purpose

File: client/termdb/tree.js (hunk 1 above)
Comment (airenzp) [reply client/termdb/tree.js:209]: I added a comment in the code. Having areference to the parent allows me to identify the domain terms and render the tree in this way

Screenshot 2024-07-25 at 3 36 55 PM

File: server/shared/termdb.usecase.js (hunk 2 above)
Comment (airenzp) [reply server/shared/termdb.usecase.js:105]: done

**Review by xzhou82 — DISMISSED** [review]

File: client/termdb/tree.js (hunk 1 above)
Comment (xzhou82) [reply client/termdb/tree.js:209]: unfortunately it's circular ref and breaks tests http://localhost:3000/testrun.html?dir=termdb&name=tree.integration

**Review by xzhou82 — APPROVED** [review]
thanks

**Pull request merged**